tapchannel/aux_closer.go

```diff
@@ -162,8 +162,8 @@ func fundingSpendWitness() lfn.Result[wire.TxWitness] {
 	)
 	ctrlBlockBytes, err := ctrlBlock.ToBytes()
 	if err != nil {
-		return lfn.Errf[wire.TxWitness]("unable to serialize control "+
-			"block: %w", err)
+		return lfn.Err[wire.TxWitness](fmt.Errorf("unable to "+
+			"serialize control block: %w", err))
```

Comment:
It seems to me that `lfn.Errf` was cleaner? Why are we going to `lfn.Err` here?

> This commit unifies the use of the lfn.Result return type as a
preparation for the next commit, where we change a bunch of
function/method signatures to use the Result type as well.

The next commit doesn't do that.

Reply:
My IDE shows an "issue" with the code otherwise: `This verb can be used only in 'fmt. Errorf' calls (%w) ` which annoys me quite a bit...
Also the use of `lnf.Err()` vs. `lnf.Errf()` wasn't really consistent.
But I'll change it back if you feel strongly about it...

Updated the commit message to reference the correct commit that introduces the changes.

Reply:
We can exclude `lnf.Errf` from that IDE check by clicking on the lightbulb:

![image](https://github.com/user-attachments/assets/a4d3ca68-3e17-46eb-9032-11df4502fd66)

It removes all formatting checks for `lnf.Errf`, but won't we only use `%w` mostly anyway perhaps.

Keeping `lfn.Errf` does seem worth it to me because we wrap errors very often in our codebase. And if we can avoid the internal `fmt.Errorf(` then I think our code would be more clear and succinct.

Reply:
Thanks for the tip. I reverted the `lnf.Err(fmt.Errorf(...))` occurrences with `lnf.Errorf(...)` again.